pkg/operator/starter.go

```diff
@@ -223,6 +224,17 @@ func prepareOauthOperator(controllerContext *controllercmd.ControllerContext, op
 
 	oauthInformers := oauthinformers.NewSharedInformerFactory(oauthClient, resync)
 
+	invalidCertsController := metricscontroller.NewMetricsController(
+		"InvalidProviderCertsController",
```

Comment:
Given that the only condition message we'll get is: `<controllerName>::InvalidCertsDetected: x server certificates without SAN detected`, should we maybe call this something else?

Reply:
let's call it `InvalidCertsController`, similar to cluster-kube-apiserver-operator we might have more metrics in here.

Reply:
the discussion results are manifested in https://github.com/openshift/library-go/pull/1289